packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -97,43 +98,6 @@ class CupertinoApp extends StatefulWidget {
     this.debugShowCheckedModeBanner = true,
   }) : assert(routes != null),
        assert(navigatorObservers != null),
-       assert(
```

Comment:
Why are these asserts no longer valid?

Reply:
This will still get asserted at runtime, just at a slightly different spot in the stack (instead of `MaterialApp`'s ctor, it'll be on `WidgetsApp`'s ctor in `MaterialApp`'s build method, for example)